CHANGELOG.md

```diff
@@ -9,12 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - **BREAKING** supervisor: actors discard `ValidateConfig` by default ([#87]).
+- **BREAKING** configurer: send `UpdateConfig` as a regular message instead of a request ([#90]).
```

Comment:
Need to double-check if this is breaking or not. I am not sure if the request token is still being generated if the message of type `Request` is send as a regular message. If not, this might break existing actors.

Reply:
@loyd I read through the code a little bit more, we might need to either change `UpdateConfig` from request to a regular message or update `msg!` to allow matching without token for request messages + special handling on the side of the requested actor to discard unmonitored replies to requests sent as a regular message.

Changing `UpdateConfig` type sounds like a way to go for me, curious what you think about it.